### ISSUE TITLE
Accessibility bug - contrast ratio on filters button does not meet accessibility requirements

### ISSUE THREAD

**Issue by eric-serrato95** [HHS/simpler-grants-gov#11481]
### Summary

The contrast ratio between filters button that triggers the side drawer and the background is not high enough, sitting at around 4.3 when it should be at least 4.5 

### Reproducibility

I tried multiple times and saw the bug every time

### How to reproduce the bug

- Check contrast ration on filter button 

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

### Acceptance Criteria

- [ ] Work with Andy in Design to figure out if we can make a site wide shift in the colors or if we need to fix this as a one off
- [ ] Update the colors to have the correct contrast ratio
